### ISSUE TITLE
bug(types): scopes is required in `firebase.login`

### ISSUE THREAD

**Issue by lazarljubenovic** [prescottprue/react-redux-firebase#671]
**Do you want to request a *feature* or report a *bug*?**  Bug

**What is the current behavior?**

Using

```ts
this.props.firebase.login({
  provider: 'google',
  type: 'popup',
})
```

results in an error.

```text
TypeScript error: Argument of type '{ provider: "google"; type: "popup"; }'
is not assignable to parameter of type 'Credentials'.
Property 'scopes' is missing in type '{ provider: "google"; type: "popup"; }'
but required in type
'{ provider: "facebook" | "google" | "twitter"; type: "popup" | "redirect"; scopes: string[]; }'.
TS2345
```

**What is the expected behavior?**

It should work according to the docs; and it _does_ work when I cast as `any`.

**Which versions of dependencies, and which browser and OS are affected by this issue? Did this work in previous versions or setups?**

Both `next` and `types-overhaul` branches seem to be affected.

**Comment by prescottprue** [comment]
Made `scopes` optional and pushed that up to `types-overhaul`. Is it working as expected now? Planning on getting these changes into a new version soon.

**Comment by lazarljubenovic** [comment]
Yup, it works now, thanks! Not sure when do we close issues (merge to next? merge to v3? merge to master?) so Imma leave that to you. :sweat_smile:

**Comment by prescottprue** [comment]
I usually close it when it has been released (either merging to master or next).

**Comment by prescottprue** [comment]
Released in v3.0.0-alpha.13
